src/SWP/Bundle/FixturesBundle/DataFixtures/ORM/LoadWidgetsData.php

```diff
@@ -26,11 +28,14 @@ class LoadWidgetsData extends AbstractFixture implements FixtureInterface, Order
      */
     public function load(ObjectManager $manager)
     {
+        /** @var TenantInterface $tenant */
```

Comment:
Tenants were stored using doctrine orm before. Any reason for the change? I think that mixing the two is not such a great idea actually. It is obviously not explicitly supported.